orchestra/google_apps/convenience.py

```diff
@@ -217,6 +223,18 @@ def upload_file(parent_id, file_path, title, description, mimetype):
         description,
         parent_id,
         mimetype,
+        mimetype,
         file_path
     )
     return file_metadata
+
+
+def get_google_spreadsheet_as_csv(gdoc_url, reader=csv.DictReader):
+    matches = GSPREAD_RE.match(gdoc_url)
+    if matches is None:
```

Comment:
Cleaned this up at your direction --- we recast this newly minted ValueError as an ImportError in the caller, so all exceptions happen cleanly without `assert`s

Reply:
ty! Will be good to unify this with our closed-source implementation in your follow-up task!